maven-java/pom.xml

```diff
@@ -117,6 +117,22 @@
     </profile>
   </profiles>
 
+  <pluginRepositories>
+    <pluginRepository>
+      <id>akka-repository</id>
+      <name>Akka library repository</name>
+      <url>https://repo.akka.io/maven</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <repositories>
+    <repository>
+      <id>akka-repository</id>
+      <name>Akka library repository</name>
+      <url>https://repo.akka.io/maven</url>
+    </repository>
+  </repositories>
```

Comment:
we have it already in the parent pom, the one used by the users.

Reply:
That is not enough, see
- https://github.com/lightbend/kalix-jvm-sdk/actions/runs/9854772888/job/27490218154#step:8:95

Reply:
Ok, I see.